haskell-src/Concordium/Types/Accounts.hs

```diff
@@ -230,9 +234,20 @@ bieBakerPoolInfo :: (AVSupportsDelegation av) => Lens' (BakerInfoEx av) BakerPoo
 bieBakerPoolInfo =
     lens _bieBakerPoolInfo (\bie x -> bie{_bieBakerPoolInfo = x})
 
+-- | Lens for '_bieBakerIsSuspended'
+{-# INLINE bieAccountIsSuspended #-}
+bieAccountIsSuspended ::
+    (AVSupportsDelegation av) =>
+    Lens' (BakerInfoEx av) (Conditionally (SupportsValidatorSuspension av) Bool)
```

Comment:
I think it would be more useful to have:
```haskell
bieAccountIsSuspended :: (AVSupportsValidatorSuspension av) => Lens' (BakerInfoEx av) Bool
```

Reply:
good idea, changed.